### PR TITLE
Refs #31 Support Django 2

### PR DESCRIPTION
**Pull request by vrialland**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,13 +1,14 @@
 [tox]
 skipsdist = False
 envlist =
-    py{27,34,35}-django{18,19,110,111},linters
+    py{27,34,35,36}-django{18,19,110,111},py{36}-django{111,20},linters
```
Comment (zebuline) [inline tox.ini]: duplicated py36-django111

File: django_north/management/commands/migrate.py [hunk 2]
```diff
@@ -16,6 +17,21 @@
...
+        parser.add_argument(
+            '--database', action='store', dest='database',
+            default=DEFAULT_DB_ALIAS,
+            help='Nominates a database to synchronize. Defaults to the "default" database.',
+        )
+        parser.add_argument(
+            '--run-syncdb', action='store_true', dest='run_syncdb',
+            help='Creates tables for apps without migrations.',
+        )
+
```
Comment (zebuline) [inline django_north/management/commands/migrate.py]: Why do you add these options ? They are not used in the following code.
Warning for the `run-syncdb` option: is it really relevant in a north context ?

**Comment by vrialland** [comment]
@zebuline Django 2.0 has added validation on every argument passed to `call_command` (https://docs.djangoproject.com/en/2.0/releases/2.0/#call-command-validates-the-options-it-receives)

Any argument that is not declared raises an error otherwise :(

File: .travis.yml [hunk 3]
```diff
@@ -43,6 +43,21 @@ matrix:
...
+
+        - python: 3.6
+          env: TOX_ENV=py36-django110
+
+        - python: 3.6
+          env: TOX_ENV=py36-django111
+
+        - python: 3.6
+          env: TOX_ENV=py36-django20
+
```
Comment (ewjoachim) [inline .travis.yml]: Not sure we have to test every combination of Python and Django, as long as we test each version once.

**Comment by vrialland** [comment]
@ewjoachim @zebuline now all tests are green, and it seems to work well with our project. I'm not very satisfied though with the handling of the parameters, can we take some time to talk about it?

**Comment by ewjoachim** [comment]
BTW, Django 1.8, 1.9 and 1.10 are not supported anymore (even extended support). Maybe it's time to drop them ?

**Comment by zebuline** [comment]
Why not

**Pull request merged**